<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Batch Image Resize</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_Source">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>8</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Destination">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>48</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Destination">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>48</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_X">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>139</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Х</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Preset">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>88</y>
      <width>401</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1280x720_90 - HD. Оптимальное качество фото.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1920x1080_90 - FullHD. Оптимальное качество фото.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048x1152_90 - 2K UHD. Оптимальное качество фото.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4096x2304_90 - 4K UHD. Оптимальное качество фото.</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_Height">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>138</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Preset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>93</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Пресет:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>178</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Quality">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>138</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Width">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>138</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Source">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Источник</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>218</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Quality">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Качество (JPEG):</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
